src/components/table/template.njk

```diff
@@ -0,0 +1,29 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if options.caption %}
+  <caption class="govuk-c-table__caption {% if options.captionSize %} {{ options.captionSize }} {% endif %}">{{ options.caption }}</caption>
+  {% endif %}
+  {% if data.head %}
+  <thead class="govuk-c-table__head">
+    <tr class="govuk-c-table__row">
+    {% for item in data.head %}
+      <th class="govuk-c-table__header {% if item.type %} govuk-c-table__header--{{ item.type }} {% endif %}" {% if item.colspan %} colspan="{{ item.colspan }}" {% endif %} {% if item.rowspan %} rowspan="{{ item.rowspan }}" {% endif %} scope="col">{{ item.text }}</th>
+    {% endfor %}
+  </tr>
+  </thead>
+  {% endif %}
+  <tbody class="govuk-c-table__body">
+    {% for rows in data.rows %}
```

Comment:
Would make more sense for this to be called `row`.

Reply:
yep, agree. done